lib/config-seed/variables.env

```diff
@@ -62,7 +62,7 @@ EXTERNAL_AUTH=none
 # SHARELATEX_TEMPLATES_USER_ID=578773160210479700917ee5
 # SHARELATEX_NEW_PROJECT_TEMPLATE_LINKS=[{"name":"All Templates","url":"/templates/all"}]
 
-TEX_LIVE_DOCKER_IMAGE=quay.io/sharelatex/texlive-full:2020.1
-ALL_TEX_LIVE_DOCKER_IMAGES=quay.io/sharelatex/texlive-full:2020.1,quay.io/sharelatex/texlive-full:2019.1
+# TEX_LIVE_DOCKER_IMAGE=quay.io/sharelatex/texlive-full:2020.1
+# ALL_TEX_LIVE_DOCKER_IMAGES=quay.io/sharelatex/texlive-full:2020.1,quay.io/sharelatex/texlive-full:2019.1
```

Comment:
Since we're already here:

```suggestion
# TEX_LIVE_DOCKER_IMAGE=quay.io/sharelatex/texlive-full:2021.1
# ALL_TEX_LIVE_DOCKER_IMAGES=quay.io/sharelatex/texlive-full:2021.1,quay.io/sharelatex/texlive-full:2020.1
```

In the past we kept untouched (and outdated) the default configurations in Server Pro to keep backwards compatibility when the user doesn't specify the versions in environment variables. But now the toolkit configuration remains untouched between updates, so it's safe to keep this version numbers up to date in the config seed.

Reply:
Thanks!